### PR TITLE
Add bilby

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for [`bilby`](https://git.ligo.org/lscsoft/bilby).

cc: @GregoryAshton

### CODE REVIEW

File: recipes/bilby/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  home: https://git.ligo.org/lscsoft/bilby
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
+  summary: A user-friendly Bayesian inference library
+
+extra:
+  recipe-maintainers:
+    - duncanmmacleod
+    - GregoryAshton
```
Comment (beckermr) [inline recipes/bilby/meta.yaml]: @GregoryAshton should confirm they would like to be a maintainer. :)

File: recipes/bilby/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - python -c "import bilby.core"
+    - python -c "import bilby.gw"
+    - python -c "import bilby.hyper"
+    - python -c "import cli_bilby"
+
+about:
+  home: https://git.ligo.org/lscsoft/bilby
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
```
Comment (beckermr) [inline recipes/bilby/meta.yaml]: It would be good to make a PR upstream to add the license file to the pypi upload.

Comment (duncanmmacleod) [reply recipes/bilby/meta.yaml]: @beckermr, see https://git.ligo.org/lscsoft/bilby/merge_requests/323

File: recipes/bilby/meta.yaml (hunk 1 above)
Comment (GregoryAshton) [reply recipes/bilby/meta.yaml]: Confirmed! Thanks kindly to @duncanmmacleod for doing this.

**Comment by duncanmmacleod** [comment]
Thanks @beckermr!

**Pull request merged**